echopype/consolidate/api.py

```diff
@@ -174,3 +182,122 @@ def sel_interp(var):
     interp_ds["longitude"] = interp_ds["longitude"].assign_attrs({"history": history})
 
     return interp_ds.drop_vars("time1")
+
+
+def add_splitbeam_angle(
+    ds: xr.Dataset,
+    echodata: EchoData,
+    waveform_mode: str,
+    encode_mode: str,
+    pulse_compression: bool = False,
+) -> xr.Dataset:
+    """
+    Add split-beam (alongship/athwartship) angles into the Sv dataset.
+    This function calculates the alongship/athwartship angle using data stored
+    in the beam groups of ``echodata``. In cases when the data does not exist,
+    an error is issued and no angle variables are added to the dataset.
+
+    Parameters
+    ----------
+    ds : xr.Dataset
+        An Sv or MVBS dataset for which the split-beam angles should be added to
```

Comment:
```suggestion
        An Sv dataset for which the split-beam angles should be added to
```
Removing the MVBS part because the angle data when first computed would be of the same dimension as the raw-resolution Sv. Also because we put this function under the `consolidate` package.

But this raises a good question of what happens when user have an Sv dataset with the splitbeam angle already added, how do we handle the mean operation in the angle data variables when running `compute_MVBS` -- since without the binned average thing, the resulting dimension of the (mean) Sv and angle data would be different after MVBS is computed.